### PR TITLE
Setup Snyk, upgrade deps, fix broken links.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: content/talks/user-unfriendly.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ events:
   - venue: Metro State Mobile Prototyping
     adr: Denver, CO
     date: 2018-04-05
-    slides: https://oddbooksapp.com/book/mobile-prototype
+    slides: #this-is-broken-@@@
```
Comment (jgerigmeyer) [inline content/talks/user-unfriendly.md:56]: @mirisuzanne This link seems to be broken. Has it moved somewhere?

File: package.json [hunk 2]
```diff
@@ -39,48 +39,48 @@
     "build": "run-s build:assets build:site eleventy",
     "update-oss-docs": "./bin/update-oss-docs",
     "prod": "ELEVENTY_ENV=production run-s update-oss-docs build",
-    "checklinks": "hyperlink --canonicalroot https://www.oddbird.net/ -r -p --skip \"_site/feed.atom\" --skip \"docs.djangoproject.com\" --skip \"/docs/\" _site/index.html",
+    "checklinks": "yarn dlx hyperlink --canonicalroot https://www.oddbird.net/ -r -p --skip \"_site/feed.atom\" --skip \"docs.djangoproject.com\" --skip \"/docs/\" --skip \"codepen.io\" --skip \"github.com\" --skip \"caniuse.com\" --skip \"youtu.be\" --skip \"stackoverflow.com/users/\" _site/index.html",
```
Comment (jerivas) [inline package.json:42]: Didn't know about this utility. Pretty cool 👍

**Review by jerivas — APPROVED** [review]

File: content/talks/user-unfriendly.md (hunk 1 above)
Comment (mirisuzanne) [reply content/talks/user-unfriendly.md:56]: I can't find it at all. We should just remove that link entirely

Comment (mirisuzanne) [reply content/talks/user-unfriendly.md:56]:
```suggestion
```

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**